### ISSUE TITLE
Socket timeout and binary encoder tests sometimes fails.

### ISSUE THREAD

**Issue by kaero** [nodules/asker#100]
https://travis-ci.org/nodules/asker/builds/43318134
https://travis-ci.org/nodules/asker/builds/43318136

**Comment by narqo** [comment]
Encoders part was fixed in #112.

**Comment by kaero** [comment]
@narqo had you observe flaky results of tests in last time?

**Comment by kaero** [comment]
@narqo ping

**Comment by narqo** [comment]
No, but as we request an external resources in our https tests (e.g. `https://yandex.com`), I don't see a reason it to be 100% stable.
